community/sample-apps/realtime-chat/hasura/metadata/databases/databases.yaml

```diff
@@ -3,7 +3,7 @@
   configuration:
     connection_info:
       database_url:
-        from_env: SAMPLE_APPS_DATABASE_URL
+        from_env: HASURA_GRAPHQL_DATABASE_URL
```

Comment:
Can we change this to `PG_DATABASE_URL` since that's the default for local dev setup and on Hasura Cloud?